src/Components/Tabs.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+interface Tab {
+  id: string,
+  title: string,
+  content: string,
+}
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: any,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  return (
+    <>
+      <ul className="Tabs__list">
+        {tabs.map(tab => (
+          <button
+            key={tab.id}
+            className="Tabs__button"
+            type="button"
+            value={tab.id}
+            onClick={(event) => {
+              const element = event.currentTarget as HTMLInputElement;
+
+              onTabSelected(element.value);
```

Comment:
why not use event.target?
```suggestion
              onTabSelected(event.target.value);
```